common/common-client/src/main/java/org/ow2/proactive/authentication/jaas.config

```diff
@@ -22,10 +22,10 @@ SchedulerIAMLoginMethod {
 	roleAttributeName="role"
 	roleSeparator=","
 	ssoTicketMarker="action"
-	jsonWebTokenSigned="false"
-	jsonWebTokenEncrypted="false"
-	jsonWebTokenSignatureKey=""
-	jsonWebTokenEncryptionKey="";
+	jsonWebTokenSigned="true"
+	jsonWebTokenEncrypted="true"
+	jsonWebTokenSignatureKey="_pyp8dL5rHqLXiS6aBzvbJ_LZdBN3-oQqbJVbC5mveTdsZ1r3WM_j3SUENB65XS7EAu7Kop3rfQx81GS_mu1Yw"
```

Comment:
this key should be hard written here?

Reply:
Yes, PA is considered as a client of IAM (and it uses JAAS for authentication), so the configuration of PA should be written in jaas.config